jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch

```diff
@@ -1,5 +1,7 @@
 <launch>
   <arg name="TRAINED_PATH" value="$(find jsk_apc2016_common)/data/trained_segmenter.pkl" />
+  <arg name="INPUT_BIN_ARRAY" value="/set_bin_param/bin_array"/>
```

Comment:
`set_bin_param` node is started in this launch file, and it depends on node name in thisfile.
thereforem we don't need to modify `arg` param.